sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -449,6 +453,8 @@ struct IsAOTForArchitectureClass {
         __SYCL_TARGET_AMD_GPU_GFX908__ == 1;
     arr[static_cast<int>(arch::amd_gpu_gfx90a)] =
         __SYCL_TARGET_AMD_GPU_GFX90A__ == 1;
+    arr[static_cast<int>(arch::amd_gpu_gfx90c)] =
+        __SYCL_TARGET_AMD_GPU_GFX90C__ == 1;
```

Comment:
Please also add a support of 
https://github.com/intel/llvm/blob/6e2d9f7144f1f6b52f077a436945c0ad9546fa16/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc#new-member-function-of-device-class
and
https://github.com/intel/llvm/blob/6e2d9f7144f1f6b52f077a436945c0ad9546fa16/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc#new-device-descriptor
for `arch::amd_gpu_gfx90c`

For that, please update `NVIDIA_AMD_ARCHES`:
https://github.com/intel/llvm/blob/974cc70f9160e7002f4dd3774ae0556ee34a4cb8/sycl/source/detail/device_info.hpp#L588

Reply:
Thanks.